hbase-server/src/main/java/org/apache/hadoop/hbase/master/LoadBalancer.java

```diff
@@ -86,6 +86,18 @@
   // We deliberately use 'localhost' so the operation will fail fast
   ServerName BOGUS_SERVER_NAME = ServerName.valueOf("localhost,1,1");
 
+  enum RunMode {
+    CHORE, REQUEST, FORCE, DRY_RUN;
+
+    boolean isForced() {
+      return this == FORCE;
+    }
+
+    boolean isDryRun() {
+      return this == DRY_RUN;
+    }
```

Comment:
IMO, these aren't adding much over `runMode == RunMode.DRY_RUN` and `runMode == RunMode.FORCE`.

Reply:
I find a function wrapper to be easier to read (and less verbose) when enum values are repeatedly compared like we are doing. This is probably personal preference though, so I've removed them.